.gitignore

```diff
@@ -117,7 +117,6 @@ dmypy.json
 # editor
 .vscode
```

Comment:
![Uploading CleanShot 2023-10-30 at 06.57.49@2x.png…]()

Reply:
And I think we can move the Rstudio bit to a place where it's less front-and-center. Not that many folks will use it.

Reply:
Should _RStudio_ queue on the user guide? At the end of the day it is a tutorial just as the others